diff --git a/src/vector/index.ts b/src/vector/index.ts
index 2010c95..0be22a1 100644
--- a/src/vector/index.ts
+++ b/src/vector/index.ts
@@ -136,25 +136,6 @@ async function start(): Promise<void> {
         // give rageshake a chance to load/fail, we don't actually assert rageshake loads, we allow it to fail if no IDB
         await settled(rageshakePromise);
 
-        const fragparts = parseQsFromFragment(window.location);
-
-        // don't try to redirect to the native apps if we're
-        // verifying a 3pid (but after we've loaded the config)
-        // or if the user is following a deep link
-        // (https://github.com/element-hq/element-web/issues/7378)
-        const preventRedirect = fragparts.params.client_secret || fragparts.location.length > 0;
-
-        if (!preventRedirect) {
-            const isIos = /iPad|iPhone|iPod/.test(navigator.userAgent) && !window.MSStream;
-            const isAndroid = /Android/.test(navigator.userAgent);
-            if (isIos || isAndroid) {
-                if (document.cookie.indexOf("element_mobile_redirect_to_guide=false") === -1) {
-                    window.location.href = "mobile_guide/";
-                    return;
-                }
-            }
-        }
-
         // set the platform for react sdk
         preparePlatform();
         // load config requires the platform to be ready
